docs/pipeline/automation/tumor_normal.md

```diff
@@ -1,8 +1,18 @@
 # Tumor Normal ICA Pipeline Lambda
 
-> NOTE:
-> * In most recent cases, you should be able to drive Tumor/Normal trigger through Portal Launch Pad UI.
-> * The following are alternates; when you need to "pair" sample manually and, like to trigger them - [ditto](https://umccr.slack.com/archives/CP356DDCH/p1698793560313359).
+_(See [automated_pipeline.svg](../../model/automated_pipeline.svg))_
+
+
+## Notes
+
+- See [metadata.md](../metadata.md) note for NTC/PTC sample treatment.
+- In most recent cases, you should be able to drive Tumor/Normal trigger through Portal Launch Pad UI.
+- The following options are alternates; when you need to "pair" sample manually and, like to trigger them - [ditto](https://umccr.slack.com/archives/CP356DDCH/p1698793560313359).
+- Pairing algorithm
+  - Topup (library having `_topup`) are merged with previous libraries of the same name
+  - No logic yet for reruns (i.e. it will skip library having `_rerun` suffix)
+  - https://github.com/umccr/data-portal-apis/pull/262
+  - https://github.com/umccr/data-portal-apis/pull/596
```

Comment:
Hi @ohofmann (and @alexiswl) Pls feel free to review this T/N notes; and below and above. Have a comment, if any. Just documenting current status quo. fyi